### DISCUSSION TITLE
Stopping a directory/folder creation

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#3169] [Q&A]
How do I stop the creation of "stellarium" and its sub-folders in the C:\Users\k\appdata\local\ folder on Win 10. v23.1

I tried the suggestion in the manual (5.1.4 and 6.2.2) with the environment variable. But that didn't seem to do anything. Also have --user-dir "e:\programs\Stellarium-new\data2" option set.

[partial log.txt](https://github.com/Stellarium/stellarium/files/11195043/partial.log.txt)

**Comment by gzotti** [comment]
Did that not fix it? 
https://github.com/Stellarium/stellarium/issues/3079

**Comment by Keatah** [comment]
I don't believe so.

This is my "Target" command line:
E:\_Programs\Stellarium-new\stellarium.exe --user-dir "e:\_programs\Stellarium-new\data2"

This is the "Start In" folder:
E:\_programs\Stellarium-new

Setting the Environment Variable STEL_USERDIR to E:\test didn't do anything either.

**Reply by gzotti** [reply]
Weird. I have STEL_USERDIR set and no AppData/Roaming/Stellarium directory.

...

Ah, AppData/Local/Stellarium! No, this cannot be changed.

**Reply by Keatah** [reply]
Ok. No problems with AppData/Roaming/Stellarium.

Can the location of the \local\ directory be made an option in a future release maybe?

**Reply by gzotti** [reply]
I saw necessity to allow the User Data dir being elsewhere when mine grew to 20GB and more (and meanwhile exceeds 100GB), on a too small C drive. However, Local/Stellarium is practically empty. So, care to explain why to move a hidden "system" directory that users should not touch elsewhere? Maybe configure Windows to move User/AppData elsewhere?

**Comment by Keatah** [comment]
I was looking to make a 100% portable install on a keydrive, with no writes to anywhere but the main stellarium folder/subfolders.

**Reply by gzotti** [reply]
There is some "portable Stellarium" published elsewhere online. Your favorite search engine helps.
